management/user/user.js

```diff
@@ -0,0 +1,102 @@
+$(function(){
+	$('#btn-add-user').on('click', function(){
+		// display new user modal
+		$('#add-user-modal').modal({});
+
+		$('#add-user-form').on('submit', function(event){
+			event.preventDefault();
+			
+			var valid = true;
+
+			$(this).find('input').each(function(){
+				var hasVal = !$(this).val();
+				if(hasVal){
```

Comment:
This hasVal variable is kinda confusing. I would've expected it to represent if the input has a value, but you're using it to represent if the input doesn't have a value. Perhaps restructure it so you have:
```var hasVal = $(this).val();
if (!hasVal) {
...
}
```

Reply:
My bad, I originally had it meaning that it had a value, and then reversed it later on without switching the name. Should be better now like `noValue`.